ui-v2/tests/acceptance/dc/acls/tokens/navigation.feature

```diff
@@ -0,0 +1,16 @@
+@setupApplicationTest
+Feature: dc / tokens / navigation
+  Scenario: Clicking a token in the listing and back again
+    Given 1 datacenter model with the value "dc-1"
+    And 3 token models
+    When I visit the tokens page for yaml
+    ---
+      dc: dc-1
+    ---
+    Then the url should be /dc-1/acls/tokens
+    And the title should be "Tokens - Consul"
+    Then I see 3 token models
+    When I click token on the tokens
+    Then the url should be /dc-1/acls/tokens/9af50b37-b545-4ef8-981b-ca94a0093d27
+    And I click "[data-test-back]"
+    Then the url should be /dc-1/acls/tokens

```

Comment:
I'd make this look exactly the same as the intention one, so don't assert the url (we just rely on the fact that this all runs without erroring), and also add the test for the create button like the intention one you just did, that way everything you've added with this is consistently being tested.